packages/neon_framework/lib/src/models/app_implementation.dart

```diff
@@ -171,7 +171,7 @@ abstract class AppImplementation<T extends Bloc, R extends AppImplementationOpti
               'assets/app.svg.vec',
               packageName: 'neon_$id',
             ),
-            semanticsLabel: NeonLocalizations.of(context).nextcloudLogo,
+            semanticsLabel: "Nextcloud Client Logo",
```

Comment:
Thanks for working on this issue but this is not what we need.
The problem is that these pictures contain the logo of the application in question. 
For example this is the files or news icon you can see in the drawer.

I think this should rather use `name(context)`.